java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -96,52 +96,54 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
 
     public static Object[] transactionTestResult() {
         return new Object[] {
-            OK,
-            value1,
-            null,
-            new String[] {value1, value2},
-            1L,
-            1L,
-            null,
-            1L,
-            null,
-            OK,
-            new String[] {value2, value1},
-            1L,
-            3L,
-            2L,
-            0L,
-            0.5,
-            1L,
-            2L,
-            value1,
-            true,
-            new String[] {value1, null, value2},
-            Map.of(field1, value1, field2, value2),
-            1L,
-            5L,
-            10.5,
-            5L,
-            5L,
-            1L,
-            OK,
-            new String[] {value3, value2},
-            value3,
-            new String[] {value2, value1},
-            3L,
-            value2,
-            new String[] {value2, value1},
-            2L,
-            1L,
-            1L,
-            Set.of("baz"),
-            3L,
-            4.0,
-            1L,
-            2L,
-            OK,
-            Map.of("timeout", "1000"),
-            OK
+            OK, // set(key1, value1)
+            value1, // get(key1)
```

Comment:
why not just add the command name like // set feels a bit too much to add the whole command.

Reply:
I can change it if you feel strongly.  This is a little easier to maintain/follow because it's more unique.

Reply:
It's not that I feel strongly about it this is was how Yurry was also doing it and my PRs are following that. Your call.